doc/intro.md

```diff
@@ -38,20 +35,20 @@ Note: Both school roll projections and small area projections take as an input p
   - [Housing-led Cohort Component Model](#housing-led-cohort-component-model)
   - [Employment-led Cohort Component Model](#employment-led-cohort-component-model)
 
-* [School roll projections](#school-roll-projections)
 
 * [Small area projections](#small-area-projections)
 
 
 ## Population projections
 
-We have chosen to implement the cohort component method as it is the most commonly used method for simple population projections.
-More complex methods aren't currently supported in `witan.models.demography`.
+We have chosen to implement the cohort component method as it is the most commonly used method for generating population projections.
+
+Alternative methods aren't currently supported in `witan.models.demography`.
 
 <pre>Cohort component population projection:
 
-This method begins with a base population, usually categorised by age and sex.
-This base population evolves through the years by applying assumptions regarding mortality, fertility and migration.
+This method begins with a starting population categorised by age and sex.
+This population iterates through the years by applying assumptions regarding mortality, fertility and migration.
```

Comment:
Replace with something like 'For each yearly iteration of the model, the population is projected by applying assumptions regarding mortality, fertility and migration.